Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -170,6 +171,16 @@ namespace Polygon_mesh_processing {
 *     \cgalParamDefault{`false`}
 *   \cgalParamNEnd
 *
+*   \cgalParamNBegin{allow_move_functor}
+*     \cgalParamDescription{A function object used to determinate if a vertex move should
+*                           be allowed or not during the relaxation step.}
+*     \cgalParamType{Unary functor that provides `bool operator()(vertex_descriptor v, Point_3 src, Point_3 tgt)`
```

Comment:
Actually, the sentence itself contradicts the word "Unary" (that would take _one_ parameter). That is not a unary functor.

Oops